hunt/app/views/animals/index.html.erb

```diff
@@ -1,18 +1,23 @@
 <ul>
-  <% @animals.each do |animal| %>
 
-    <div class="container">
-      <div class="row">
-        <div class="col-sm">
-          <li>Name: <%= link_to animal.name, animal_path(animal)  %></li>
-        </div>
-        <div class="col-sm">
-          State: <%=animal.state%>
-        </div>
-        <div class="col-sm">
-          Category: <%=animal.category %>
-        </div>
-      </div>
-  </div>
-  <% end %>
+  <div class="container">
+    <%= link_to 'New Animal', new_animal_path, class: 'btn btn-dark' %>
```

Comment:
What do you think about adding some margin spacing to the bottom of this button in order to separate it from the table? Right now they blend together a little bit. With bootstrap utility classes you should be able to add `mb-1`